README.md

```diff
@@ -7,26 +7,42 @@
 
 Fastify Redis connection plugin; with this you can share the same Redis connection in every part of your server.
 
-Under the hood [ioredis](https://github.com/luin/ioredis) is used as client, the ``options`` that you pass to `register` will be passed to the Redis client.
-
 ## Install
+
 ```
 npm i fastify-redis --save
 ```
+
 ## Usage
+
 Add it to your project with `register` and you are done!
-You can access the *Redis* client via `fastify.redis`. The client is
-automatically closed when the fastify instance is closed.
 
-```js
-'use strict'
+### Create a new Redis Client
+
+Under the hood [ioredis](https://github.com/luin/ioredis) is used as client, the ``options`` that you pass to `register` will be passed to the Redis client.
 
+```js
 const fastify = require('fastify')()
 
+// minimum options
 fastify.register(require('fastify-redis'), { host: '127.0.0.1' })
-// or
-fastify.register(require('fastify-redis'), { url: 'redis://127.0.0.1', /* other redis options */ })
```

Comment:
The `url` option is valid too. It should not be removed from the documentation examples.

Can you add it back with a dedicated example ?
And maybe adding a note saying that passing a redis url is another valid way of registering the plugin.

Reply:
Done. Has to reword the comments. Let me know if the wording works for you.